.github/workflows/ci.yml

```diff
@@ -77,13 +77,17 @@ jobs:
       run: cargo install cargo-apk
 
     - name: Build
-      if: contains(matrix.platform.target, 'android') == false
+      if: contains(matrix.platform.target, 'android') == false && contains(matrix.platform.target, 'wasm') == false
       run: cargo build --workspace --all-features --target ${{ matrix.platform.target }}
 
     - name: Build android
       if: contains(matrix.platform.target, 'android')
       run: cargo apk --target ${{ matrix.platform.target }} build --workspace --all-features
 
+    - name: Build WASM
+      if: contains(matrix.platform.target, 'wasm')
+      run: cargo build --workspace --features default,dag-cbor,dag-json,dag-pb,derive,serde-codec --target ${{ matrix.platform.target }}
```

Comment:
I think you've missed the `std` feature.

Reply:
Added